lib/game.rb

```diff
@@ -0,0 +1,118 @@
+require './lib/cell'
+require './lib/ship'
+require './lib/player'
+require './lib/board'
+require 'pry'
+
+class Game
+
+  attr_reader :computer,
+              :human,
+              :valid
+
+  def initialize
+    @computer = Player.new('computer')
+    @human = Player.new('human')
+  end
+
+  # def welcome
+  #
+  #   puts "Welcome to BATTLESHIP"
+  #   puts "Enter p to play. Enter q to quit"
+  #   input = gets.chomp
+  #
+  #
+  #     if input == "p" || input == "P"
+  #       self.start_game
+  #     elsif input == 'q' || input == "Q"
+  #       exit
+  #     else
+  #       puts 'INVALID INPUT'
+  #       self.welcome
+  #     end
+  # end
+
+  def start_game
+    @computer = Player.new(:computer)
+    @player = Player.new(:human)
+
+
+  end
+
+  def setup
+    place_ships_ai
+    place_ships_user
+  end
+
+  def place_ships_user
```

Comment:
let's refactor this later to do it the same way we did the place_ships_ai, iterating through each of the ships